controller/search.go

```diff
@@ -129,3 +131,65 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		return ctx.OK(&response)
 	})
 }
+
+// Users runs the user search action.
+func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
+
+	q := ctx.Q
+	if q == "" {
+		return ctx.BadRequest(goa.ErrBadRequest(fmt.Errorf("empty search query not allowed")))
+	} else if q == "*" {
+		q = "" // Allow empty query if * specified
```

Comment:
I don't think we should allow an empty query at all. There should not be a public way to get all the users/identities of the system.